### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the "Search" Button

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#19]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

Testing Environment:
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in EDGE.
2. Go to the search button available in the "Search Docs" search bar (with mouse as keyboard is not accessible )
3. Observe the "Name" property in Inspect

Same issue is also observed with the links
[https://code.visualstudio.com/docs
](https://code.visualstudio.com/docs
)https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/runtimes/nodejs
https://code.visualstudio.com/docs/languages/overview

**Actual Result:**
The "Name" property is empty
**_Narrator Behaviour:_**
The Narrator announces as "Button"

**Expected Result:**
The "Name" property should be specific
Eg: "Search Docs Button"

**User Impact:**
Visually impaired users  will not get to know the purpose of the menu items as the screen reader will not announce anything when the user arrives at the menu items
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![name property is empty in inspect](https://cloud.githubusercontent.com/assets/24825441/21680971/d77d8470-d372-11e6-81be-0028079667a1.png)
![narrator buddy screenshot](https://cloud.githubusercontent.com/assets/24825441/21680970/d7798c44-d372-11e6-9ae1-fa4e2723b2a8.png)

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-website/commit/72d8a13196c964b0f89a7a1915f75d7e2c9c9b90
